### PR TITLE
en,zh: add resolved-ts configuration to EBS backup limitations

### PR DESCRIPTION
**Pull request by WangLe1321**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-tidb-operator) that's required for repo owners to accept my contribution.

### Which TiDB Operator version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v1.5 (TiDB Operator 1.5 versions)
- [ ] v1.4 (TiDB Operator 1.4 versions)
- [ ] v1.3 (TiDB Operator 1.3 versions)
- [ ] v1.2 (TiDB Operator 1.2 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):

### CODE REVIEW

**Comment by lilin90** [comment]
I updated wording and value format of configuration parameters, and added links in this commit https://github.com/pingcap/docs-tidb-operator/pull/2425/commits/77d8fdab614462da4ad05ab6321fb7626c599a11.

File: zh/backup-to-aws-s3-by-snapshot.md [hunk 1]
```diff
@@ -23,6 +23,8 @@ summary: 介绍如何基于 EBS 卷快照使用 TiDB Operator 备份 TiDB 集群
 ### 使用限制
 
 - 要使用此功能，TiDB Operator 应为 v1.4.0 及以上，TiDB 应为 v6.3.0 及以上。
+- TiKV 配置中，**不能**将 [`resolved-ts.enable`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#enable-2) 设置为 `false`，也**不能**将 [`raftstore.report-min-resolved-ts-interval`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#report-min-resolved-ts-interval-%E4%BB%8E-v600-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) 设置为 `"0s"`，否则会导致备份失败。
+- PD 配置中，**不能**将 [`pd-server.min-resolved-ts-persistence-interval`](https://docs.pingcap.com/zh/tidb/stable/pd-configuration-file#min-resolved-ts-persistence-interval-%E4%BB%8E-v600-%E7%89%88%E6%9C%AC%E5%BC%80%E5%A7%8B%E5%BC%95%E5%85%A5) 设置为 `"0s"`，否则会导致备份失败。
```
Comment (Oreoxmt) [inline zh/backup-to-aws-s3-by-snapshot.md:27]:
```suggestion
- TiKV 配置中，**不能**将 [`resolved-ts.enable`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#enable-2) 设置为 `false`，也**不能**将 [`raftstore.report-min-resolved-ts-interval`](https://docs.pingcap.com/zh/tidb/stable/tikv-configuration-file#report-min-resolved-ts-interval-从-v600-版本开始引入) 设置为 `"0s"`，否则会导致备份失败。
- PD 配置中，**不能**将 [`pd-server.min-resolved-ts-persistence-interval`](https://docs.pingcap.com/zh/tidb/stable/pd-configuration-file#min-resolved-ts-persistence-interval-从-v600-版本开始引入) 设置为 `"0s"`，否则会导致备份失败。
```

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**